btb/examples/selection_example.py

```diff
@@ -0,0 +1,168 @@
+from btb import HyperParameter, ParamTypes
+from btb.selection import Selector
+from btb.tuning import GP
+import numpy as np
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.datasets import fetch_mldata
+from sklearn.svm import SVC
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
+import numpy as np
+
+
+"""
+Selection example of chosing whether to give tuning budget to a Random Foreset
+pipeline or an SVM pipeline.
+We use a Selector to decide whether to tune the RF or SVM pipeline next.
+We use a GP-based tuner for both pipelines.
+
+We tune the n_estimators and max_depth parameters of the Random Forrest.
+We tune tge c and gamma parameters of the SVM.
+"""
+
+
+def tune_pipeline(
```

Comment:
This would be better set in a single line, since it does not go past 80 characters.